projects/1.16/assets/infernal-expansion/infernalexp/lang/zh_cn.json

```
@@ -1 +1,345 @@
-{}
\ No newline at end of file
+{
+  "itemGroup.InfernalTab": "地狱扩展",
+  "block.infernalexp.dimstone": "暗荧石",
+  "block.infernalexp.dullstone": "无光荧石",
+  "block.infernalexp.smooth_glowstone": "平滑荧石",
+  "block.infernalexp.smooth_dimstone": "平滑暗荧石",
+  "block.infernalexp.smooth_dullstone": "平滑无光荧石",
+  "block.infernalexp.glowstone_bricks": "荧石砖",
+  "block.infernalexp.dimstone_bricks": "暗荧石砖",
+  "block.infernalexp.dullstone_bricks": "无光荧石砖",
+  "block.infernalexp.cracked_glowstone_bricks": "裂纹荧石砖",
+  "block.infernalexp.cracked_dimstone_bricks": "裂纹暗荧石砖",
+  "block.infernalexp.cracked_dullstone_bricks": "裂纹无光荧石砖",
+  "block.infernalexp.chiseled_glowstone_bricks": "錾制荧石砖",
+  "block.infernalexp.chiseled_dimstone_bricks": "錾制暗荧石砖",
+  "block.infernalexp.chiseled_dullstone_bricks": "錾制无光荧石砖",
+  "block.infernalexp.smooth_glowstone_slab": "平滑荧石台阶",
+  "block.infernalexp.smooth_glowstone_vertical_slab": "平滑荧石竖台阶",
```

```suggestion
  "block.infernalexp.smooth_glowstone_vertical_slab": "平滑荧石竖直台阶",
```
如果是 Quark 联动内容，应为竖直台阶，其他的也一样